### PR TITLE
chore: apply code style rules to javascript

### PR DESCRIPTION
**Pull request by JustinBeckwith**
Addresses #228

### CODE REVIEW

File: examples/adc.js [hunk 1]
```diff
@@ -43,27 +42,29 @@ async function main() {
...
  * Instead of specifying the type of client you'd like to use (JWT, OAuth2, etc)
- * this library will automatically choose the right client based on the environment.
+ * this library will automatically choose the right client based on the
+ * environment.
  */
-async function getADC() {
-  // Acquire a client and the projectId based on the environment. This method looks
-  // for the GCLOUD_PROJECT and GOOGLE_APPLICATION_CREDENTIALS environment variables.
+async function
+getADC() {
```
Comment (jinwoo) [inline examples/adc.js]: It looks quite weird to me to split `async function` and the function name across two lines, but if this is what the tool does, ok.

File: examples/fromJSON.js [hunk 2]
```diff
@@ -33,8 +33,9 @@ const {GoogleAuth} = require('../build/src/index');
  *       "client_id": "your-client-id",
  *       "auth_uri": "https://accounts.google.com/o/oauth2/auth",
  *       "token_uri": "https://accounts.google.com/o/oauth2/token",
- *       "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
- *       "client_x509_cert_url": "your-cert-url"
+ *       "auth_provider_x509_cert_url":
+ *"https://www.googleapis.com/oauth2/v1/certs", "client_x509_cert_url":
```
Comment (jinwoo) [inline examples/fromJSON.js]: This doesn't look ok to me though. Is there any way to convince clang-format otherwise?

Comment (JustinBeckwith) [reply examples/fromJSON.js]: Yeah I have no idea what clang-format is trying to do here.  I kind of wonder out loud if it supports the `async` keyword....

Comment (jinwoo) [reply examples/fromJSON.js]: Doesn't seem related to `async`. The same thing happens without `async` too. See https://github.com/google/google-auth-library-nodejs/pull/233/files#diff-7538725028be787890f7305df2148f61R43

**Comment by JustinBeckwith** [comment]
@jinwoo this is ready for another look.  I switched over to prettier, which seems to work way better.

**Comment by JustinBeckwith** [comment]
CI failures are unrelated, and discussed in #236

**Pull request merged**